pkg/utils/k8sutil.go

```diff
@@ -611,7 +612,7 @@ func DeployKubeless(client *kubernetes.Clientset, ctlImage string, ctlNamespace
 // DeployMsgBroker deploys kafka-controller
 func DeployMsgBroker(client *kubernetes.Clientset, kafkaVer string, ctlNamespace string) error {
 	labels := map[string]string{
-		"app": "kafka",
+		"function": "kafka-controller",
```

Comment:
Do we want kafka-controller to look like and (potentially) conflict with a user-provided function called "kafka-controller"?

If this is just so you can reuse `GetPods()` for the kafka-controller case, consider doing a special-case / more-specific search using a different label (perhaps `GetPods()` should wrap a more generic `GetPodsByLabel` function).